packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -293,7 +293,11 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 	const [ nonDefaultPaidPlan, setNonDefaultPaidPlan ] = React.useState< Plan | undefined >();
 
 	React.useEffect( () => {
-		if ( selectedPaidPlan && selectedPaidPlan?.storeSlug !== defaultPaidPlan?.storeSlug ) {
+		if (
+			selectedPaidPlan &&
+			defaultPaidPlan &&
+			selectedPaidPlan?.storeSlug !== defaultPaidPlan?.storeSlug
```

Comment:
This check is redundant now.

```suggestion
			selectedPaidPlan.storeSlug !== defaultPaidPlan.storeSlug
```